database/migrations/20200209204804_application_submissions.js

```diff
@@ -0,0 +1,28 @@
+exports.up = function(knex) {
+  return knex.schema.createTable("application_submissions", function(table) {
+    table.increments("id");
+    table
+      .integer("skilled_impact_request_id")
+      .references("id")
+      .inTable("skilled_impact_requests")
```

Comment:
you should be able to specify the reference as `skilled_impact_requests.id` e.g. [this reference](https://github.com/Lambda-School-Labs/key-conservation-be/blob/skills/database/migrations/20200203151942_skills.js#L55) from David's PR